index.html

```diff
@@ -143,7 +143,19 @@ <h3 class="social-media">LET’S CONNECT</h3>
         </ul>
       </section>
     </section>
-    
+    <section class="form-section">
+      <h2>Contact me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <form action="https://formspree.io/f/xzbqzgww" class="info-form" method="post">
+        <input type="text" placeholder="Yeremias" required class="input-text" maxlength="30">
+        <input type="email" placeholder="natajayanj14@gmail.com" class="input-text" maxlength="30" required>
```

Comment:
[OPTIONAL] Good job trying to add an extra level of validation to this field but there is one thing I would like to point out here and it has to do with the scenario where a contact might have a mail that is longer than 30 characters, the max length validation will become an issue for that contact and as such, they'd have a bad UX while trying to contact you.
So I think it will be better to remove that validation option.